Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -39,6 +39,7 @@
 INCIDENTS_PER_FETCH = int(demisto.params().get('incidents_per_fetch', 15))
 # Remove proxy if not set to true in params
 handle_proxy()
+ENDPOINT_CONTEXT_PATH = 'Endpoint(val.ID && val.ID == obj.ID)'
```

Comment:
you have it in the endpoint class already.

Reply:
it's needed in order to get the context out of the dict and create hr - example is in line 1656 in this file